src/ballet/keccak256/fd_keccak256_test_vector.c

```diff
@@ -0,0 +1,43 @@
+  { NULL, /* Terminate the list of vectors */
```

Comment:
I had many more vectors and I trimmed until it would compile under with-brutality. I was also aware of this limit but I have seen some other long strings in the repo.